dockerfiles/stardog.single.properties

```diff
@@ -1,2 +1,3 @@
 # Empty as we currently don't need specific properties for single node testing
-# we still need an existing property file so that the Dockerfile doesn't break
\ No newline at end of file
+# we still need an existing property file so that the Dockerfile doesn't break
+catalog.auto.create.db=false
```

Comment:
We probably don't need to disable the catalog on a standalone Stardog instance. However, if you want to remove the catalog from all instances we use in this test suite, then the comment above should be updated, since this file is no longer empty.

Reply:
right, i reverted this change.